src/rooms.js

```diff
@@ -0,0 +1,11 @@
+function findAvailableRoomsByDate(date, bookings, rooms) {
+  const alreadyBookedRoomsOnDate = bookings.filter(booking => booking.date === date);
+  const alreadyBookedRoomNumbers = alreadyBookedRoomsOnDate.map(booking => booking.roomNumber);
+  const availableRooms = rooms.filter(room => !alreadyBookedRoomNumbers.includes(room.number));
+  return availableRooms;
+}
+
+function filterRoomsByRoomType(rooms, roomType) {
+  const filteredRooms = rooms.filter(room => room.roomType === roomType);
```

Comment:
Similarly to the comment above, you could return rooms.filter on line 9 and delete line 10